framework/aster-frame/src/sync/rcu/mod.rs

```diff
@@ -32,13 +90,38 @@ impl<P: OwnerPtr> Rcu<P> {
         }
     }
 
+    /// Retrieves a read guard for the RCU mechanism.
+    ///
+    /// This method returns a `RcuReadGuard` which allows read-only access to the
+    /// underlying data protected by the RCU mechanism.   
+    ///
+    /// This function has the semantics of _subscribe_ in RCU mechanism.
+    ///
+    /// # Safety
+    ///
+    /// The pointer protected by the Rcu must be valid and point to a valid object.
+    ///
+    /// # Non-Preemptible RCU
+    ///
+    /// In non-preemptible RCU, the read-side critical section is delimited using
+    /// a `PreemptGuard`.
```

Comment:
Our current implementation is non-preemptible RCU. To avoid repeating "Non-Preemptible RCU" in each method, I think it should be moved to the type-level Rust doc.